Git.hub/Client.cs

```diff
@@ -172,11 +191,11 @@ public List<Repository> searchRepositories(string query)
             var request = new RestRequest("/legacy/repos/search/{query}");
             request.AddUrlSegment("query", query);
 
-            var repos = client.Get<APIv2.RepositoryListV2>(request).Data;
-            if (repos == null || repos.Repositories == null)
-                throw new Exception(string.Format("Could not search for {0}", query));
+            var repos = DoRequest<APIv2.RepositoryListV2>(request);
+            if (repos?.Repositories == null)
```

Comment:
Maybe add {} to conform with GE style (but will not be the same as other files in this submodule)